docs/vagrant-spk/packaging-tutorial-meteor.md

```diff
@@ -224,8 +204,6 @@ but it does mean that the domain name running Sandstorm needs
 DNS. You can rest assured that your interactions with
 `local.sandstorm.io` stay entirely on your computer.
 
-<!--(**Editor's note**: We should make localhost:6080 work, so that people don't have to learn about `local.sandstorm.io`.)-->
```

Comment:
I think this is still relevant, just the port number needs updating too.

Reply:
We have this issue tracked as https://github.com/sandstorm-io/vagrant-spk/issues/73 and I marked it as a goal for vagrant-spk 1.1. I figured we should delete these because invisible comments in the docs page is not a good place to track the issue.

Reply:
Works for me; as long as we're still tracking it.